test/e2e/images_test.go

```diff
@@ -152,10 +152,10 @@ var _ = Describe("Podman images", func() {
 
 	It("podman images filter reference", func() {
 		podmanTest.RestoreAllArtifacts()
-		result := podmanTest.PodmanNoCache([]string{"images", "-q", "-f", "reference=docker.io*"})
+		result := podmanTest.PodmanNoCache([]string{"images", "-q", "-f", "reference=quay.io*"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(len(result.OutputToStringArray())).To(Equal(2))
+		Expect(len(result.OutputToStringArray())).To(Equal(3))
```

Comment:
Because `nginx` was already fetched from quay; and now alpine+busybox are as well.